src/reducers/__tests__/articleReducers.test.js

```diff
@@ -0,0 +1,67 @@
+import actionTypes from '../../actions/types';
+import articleReducer from '../articleReducers';
+
+describe('articleReducer', () => {
+  let initialState;
```

Comment:
You can actually initialize in a single line like:  
let initialState, articleData, payload;